net-misc/mediatomb/mediatomb-0.12.1.recipe

```diff
@@ -1,31 +1,85 @@
+SUMMARY="Open source UPnP Media Server"
 DESCRIPTION="MediaTomb is an open source UPnP MediaServer"
 HOMEPAGE="http://mediatomb.cc/"
```

Comment:
The `HOMEPAGE` is currently down I think, got error 1001 from CloudFlare

Reply:
Yes, seems like it's removed from DNS services?

Reply:
Should we change it to

    https://sourceforge.net/projects/mediatomb/

instead? At least that link is still alive.

Reply:
Changed :)